fstab.yaml

```
@@ -1,3 +1,6 @@
 mountpoints:
   /:
     url: https://adobe.sharepoint.com/sites/CC-Express/Shared%20Documents/website
+
+folders:
+  /express/templates/: /express/templates/default
```

We'll need to map the metadata sheet under `/express`:
```suggestion
  /express/templates/content.json: /metadata.json
  /express/templates/: /express/templates/default
```